packages/components/src/locales/en.ts

```diff
@@ -33,4 +33,5 @@ export default {
 	'error-list': 'Error list',
 	'error-list-message': 'Please correct the following errors',
 	version: 'Version number',
+	'entries-display-label': 'Entries {{start}} to {{end}} of {{total}}',
```

Comment:
Vorschlag, um zu verdeutlichen, wofür der Text steht:
```suggestion
	'table-visible-range': 'Entries {{start}} to {{end}} of {{total}}',
```